app/client-v2/src/config.ts

```diff
@@ -1,7 +1,9 @@
 export const getApiUrl = () => {
   if (import.meta.env.MODE === "development") {
     return "http://localhost:4000";
-  } else {
-    return "https://${window.location.host}/api";
   }
+  if (import.meta.env.MODE === "test") {
+    return "";
+  }
```

Comment:
I thought most of the tests for urls were for `stringContaining` so do you need this? Or is it for the mock server part?

Reply:
yeah i did but just changed to make things simpler to match in general.. so future tests and stuff